Breeze.BreezeServer/BreezeConfiguration.cs

```diff
@@ -140,7 +155,13 @@ public BreezeConfiguration(string configPath, string datadir = null)
 
                 string bitcoinNetwork;
                 
-                if (TumblerNetwork == Network.Main)
+                if (TumblerNetwork == Network.StratisMain)
+                    bitcoinNetwork = "StratisMainNet";
+                else if (TumblerNetwork == Network.StratisTest)
+                    bitcoinNetwork = "StratisTest";
+                else if (TumblerNetwork == Network.StratisRegTest)
+                    bitcoinNetwork = "StratisRegTest";
+                else if (TumblerNetwork == Network.Main)

```

Comment:
Are we able to get the network name as a string in a cleaner way than checking against every type?